examples/getting-started-movielens/04-Triton-Inference-with-HugeCTR.ipynb

```diff
@@ -55,29 +55,42 @@
    "id": "71304a10",
```

Comment:
<p>remove this line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>done</p>